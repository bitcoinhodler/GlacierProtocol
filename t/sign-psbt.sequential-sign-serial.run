#!/bin/bash
set -e

# Test large withdrawal transactions. This spends 7 inputs from an older
# Glacier address with uncompressed, non-segwit keys.


# Simulate an earlier, even-larger transaction. These files should
# get deleted by glacierscript.
touch transaction-01.png transaction-02.png transaction-03.png


$GLACIERSCRIPT --regtest=$1 sign-psbt << INPUT 2>&1
y
y
y
y
y
y
../../t/sign-psbt.sequential-sign-serial.psbt
y
2
93VDtjySSuPjgLYyCR6BBTqCuaqu6X728ZfkNEDbgAdVURRd1rS
924WjH7yMYyuyhUougBHEu2vThcHRmahF6FsKvNVQBwSGuEyqKm
y
INPUT
