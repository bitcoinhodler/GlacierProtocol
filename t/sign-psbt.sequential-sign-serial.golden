Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?Input a filename located in the current directory which contains the PSBT:
0.07230000 BTC in unspent inputs from cold storage address 2MyCc3oEVwSYzgmspKTjZ6HZrzeg8ajdmKR
0.07226753 BTC going to destination address bcrt1qrwxllgvtc0ns624dlped894kkfq608jmye2k50
Fee amount: 0.00003247
Confirm? (y/n): 
How many private keys will you be signing this transaction with (at least 2 required to complete transaction)?
#: Output will be a partially signed bitcoin transaction (PSBT).
Key #1: 
Is this data correct?
*** WARNING: Incorrect data may lead to loss of funds ***

0.07230000 BTC in unspent inputs from cold storage address 2MyCc3oEVwSYzgmspKTjZ6HZrzeg8ajdmKR
0.07226753 BTC going to destination address bcrt1qrwxllgvtc0ns624dlped894kkfq608jmye2k50
Fee amount: 0.00003247

Signing with private keys: 
93VDtjySSuPjgLYyCR6BBTqCuaqu6X728ZfkNEDbgAdVURRd1rS


Confirm? (y/n): 
Calculating transaction...

Incomplete PSBT (base64):
cHNidP8BAP1IAQIAAAAHb+LA5yHU1k5EaERAws2C3JFuZHte0GuP9vb4X9f3fW8BAAAAAP////8zWExLrIIhHesoitJAKWsjcgFKYUHdQUzGGZDeKYE9jgEAAAAA/////+5LQ1Lq95RDvjPF5hF1jJ0yaYUcpPJR2/tDpjkpxesQAQAAAAD/////wPKne7w/8Ff+l/q91SU2Gm8OVntmQAhl5lD24IICE64AAAAAAP/////DaoCSqRDkVq+L8DJkCorumodIqEigqvEV3wnREN89gQEAAAAA/////42m7kWJ+OT3bl2nJ04B/cpm1tMV62/MF5/Ir4SP2jczAAAAAAD/////+FI7r73yJsRHs1t18fRCafKxc5r4z0dmQrvRh6glNjgAAAAAAP////8BgUVuAAAAAAAWABQbjf+hi8PnDSqt+HLTlrayQaeeWwAAAAAAAQBzAgAAAAEu4nJDgFznn2k7/5HcW9iDYkM89vhpVz9cJHhsMljj7wEAAAAA/////wJC5BkAAAAAABepFOqk58X9mt7GJc65A6r5b7pqYZtvh0BCDwAAAAAAF6kUQVHsDJUyftyhxMXTjQHgDZ7JPVeHAAAAAEICBP8YihMe55kApM2+aMXqhK1zuKz4QU5YlfbHsOR3nvUNObo2mjTNP3IWh3dpopLM9SwmU8eZoTgyG/jB4DqRIFlHMEQCIEHXyWsSPlPU+ANmWjTOVxk6u5JgNSUMeWCIR/Hr+uERAiA7tEEvGs3vGbyRw4K4FQU6QakWZBIb9mYH451wHmW+AAEBBP0LAVJBBJgTe2Ijqfo5Xhb+bkmn9Kc3Ap1Eduw1hEqEaeNY1kG4WlvzVdVwaZ1jzeh9odRQ4ZZtl+5hqZtRmVoR636g9nRBBP2aafleMcO40sl5JeDxUd0cZ/O3Hjum248g2UTm6U2bXjrUwSpfm9bSE3z2mwFfljfeYar64p/fzPjvWkzYHjdBBP8YihMe55kApM2+aMXqhK1zuKz4QU5YlfbHsOR3nvUNObo2mjTNP3IWh3dpopLM9SwmU8eZoTgyG/jB4DqRIFlBBO6vwGIQCGYqmXe5U+5/evh5gYMPVhysUDhyEcddWLy3UUFPyavgz0oopARIBszVSz+AupkODv8Ea7G5MJaLlzpUrgABAIoCAAAAAdf02skDq07Rhq89vCY+buAGPPCtM6/0ffb0nP73eYgPAQAAABcWABR1g0EXPduZd4r4qQ8aYoaaMD6xHv////8ChmByAAAAAAAXqRRpKKmqeVVhIxwxL+ZkOu2iBt2rpIdAQg8AAAAAABepFEFR7AyVMn7cocTF040B4A2eyT1XhwAAAABCAgT/GIoTHueZAKTNvmjF6oStc7is+EFOWJX2x7Dkd571DTm6Npo0zT9yFod3aaKSzPUsJlPHmaE4Mhv4weA6kSBZRzBEAiATW53pBpt0wkEo2g/AlVifdDJ8Jk6FL6ig1NidV6h8ggIgEyovZFsCPaSHipKa9al6iJELybNXTCrGPFZbTitdmN4BAQT9CwFSQQSYE3tiI6n6OV4W/m5Jp/SnNwKdRHbsNYRKhGnjWNZBuFpb81XVcGmdY83ofaHUUOGWbZfuYambUZlaEet+oPZ0QQT9mmn5XjHDuNLJeSXg8VHdHGfztx47ptuPINlE5ulNm1461MEqX5vW0hN89psBX5Y33mGq+uKf38z471pM2B43QQT/GIoTHueZAKTNvmjF6oStc7is+EFOWJX2x7Dkd571DTm6Npo0zT9yFod3aaKSzPUsJlPHmaE4Mhv4weA6kSBZQQTur8BiEAhmKpl3uVPuf3r4eYGDD1YcrFA4chHHXVi8t1FBT8mr4M9KKKQESAbM1Us/gLqZDg7/BGuxuTCWi5c6VK4AAQBzAgAAAAGosKEp0chiEitYzaITv46mwp9CIjyqST5/W8lFBx0fPgEAAAAA/////wKQxMABAAAAABepFJxqaUEKWTf81O/eDvynLBkKspr1h1BpDwAAAAAAF6kUQVHsDJUyftyhxMXTjQHgDZ7JPVeHAAAAAEICBP8YihMe55kApM2+aMXqhK1zuKz4QU5YlfbHsOR3nvUNObo2mjTNP3IWh3dpopLM9SwmU8eZoTgyG/jB4DqRIFlHMEQCIAFNco44+HYHLW+Dg2bx39AO5hNPPnTr/wdcrKaOLIBEAiBLlr/xC1l3iJ8Vc1a7l4GjLmCD6hmcwsXuZneON/WAvQEBBP0LAVJBBJgTe2Ijqfo5Xhb+bkmn9Kc3Ap1Eduw1hEqEaeNY1kG4WlvzVdVwaZ1jzeh9odRQ4ZZtl+5hqZtRmVoR636g9nRBBP2aafleMcO40sl5JeDxUd0cZ/O3Hjum248g2UTm6U2bXjrUwSpfm9bSE3z2mwFfljfeYar64p/fzPjvWkzYHjdBBP8YihMe55kApM2+aMXqhK1zuKz4QU5YlfbHsOR3nvUNObo2mjTNP3IWh3dpopLM9SwmU8eZoTgyG/jB4DqRIFlBBO6vwGIQCGYqmXe5U+5/evh5gYMPVhysUDhyEcddWLy3UUFPyavgz0oopARIBszVSz+AupkODv8Ea7G5MJaLlzpUrgABAIoCAAAAAT+qiMRWCgO3hj3DvzzFUBUHdn4rGCD+BeLBl5klDn4wAQAAABcWABQAYvCbVGI/TpTIQkT79vyRf8MSrv////8CgN4PAAAAAAAXqRRBUewMlTJ+3KHExdONAeANnsk9V4dggWIAAAAAABepFB+tzUK0nKXb5JmJmqnLmnIoYzgJhwAAAABCAgT/GIoTHueZAKTNvmjF6oStc7is+EFOWJX2x7Dkd571DTm6Npo0zT9yFod3aaKSzPUsJlPHmaE4Mhv4weA6kSBZRzBEAiBNRWua5pl3FRWqQuGJOs5gdEMeHIxNrya3Jz6uYRDMswIgaIEEVK7jmWgPielFIAOqBC2NVV5hc4pQSV1siJJ0FmkBAQT9CwFSQQSYE3tiI6n6OV4W/m5Jp/SnNwKdRHbsNYRKhGnjWNZBuFpb81XVcGmdY83ofaHUUOGWbZfuYambUZlaEet+oPZ0QQT9mmn5XjHDuNLJeSXg8VHdHGfztx47ptuPINlE5ulNm1461MEqX5vW0hN89psBX5Y33mGq+uKf38z471pM2B43QQT/GIoTHueZAKTNvmjF6oStc7is+EFOWJX2x7Dkd571DTm6Npo0zT9yFod3aaKSzPUsJlPHmaE4Mhv4weA6kSBZQQTur8BiEAhmKpl3uVPuf3r4eYGDD1YcrFA4chHHXVi8t1FBT8mr4M9KKKQESAbM1Us/gLqZDg7/BGuxuTCWi5c6VK4AAQCKAgAAAAGHeEwhVVwNW4pqD4ImyV/0Ad766e3KMxU+yZw9+ZlirAEAAAAXFgAULZcdgnHvBqHaM2TIPvG7CZvojPv/////Aip7UgAAAAAAF6kUvTl7YUKhZatUtF/rp+CGz3MjtECHkAUQAAAAAAAXqRRBUewMlTJ+3KHExdONAeANnsk9V4cAAAAAQgIE/xiKEx7nmQCkzb5oxeqErXO4rPhBTliV9sew5Hee9Q05ujaaNM0/chaHd2miksz1LCZTx5mhODIb+MHgOpEgWUcwRAIgBGTi6qRv+SqTe9pFmrJeZnUAS8GRInV7RxP4lP+ssUACIEEBtkb0oUD9b6b6xP4Xh7mB6rlxGEop3cJbZidpMbT/AQEE/QsBUkEEmBN7YiOp+jleFv5uSaf0pzcCnUR27DWESoRp41jWQbhaW/NV1XBpnWPN6H2h1FDhlm2X7mGpm1GZWhHrfqD2dEEE/Zpp+V4xw7jSyXkl4PFR3Rxn87ceO6bbjyDZRObpTZteOtTBKl+b1tITfPabAV+WN95hqvrin9/M+O9aTNgeN0EE/xiKEx7nmQCkzb5oxeqErXO4rPhBTliV9sew5Hee9Q05ujaaNM0/chaHd2miksz1LCZTx5mhODIb+MHgOpEgWUEE7q/AYhAIZiqZd7lT7n96+HmBgw9WHKxQOHIRx11YvLdRQU/Jq+DPSiikBEgGzNVLP4C6mQ4O/wRrsbkwlouXOlSuAAEAigIAAAAB5+YDru5Tl++YlRQLg9IGL1Ea9YD291fKr1HyzKtHhNABAAAAFxYAFFsl3j9Y0jN1MNTlnKG278o4xJxh/////wKgLBAAAAAAABepFEFR7AyVMn7cocTF040B4A2eyT1Xh0qXsAEAAAAAF6kUHUWriQ+pt1TDiiBlarBjV1fBW52HAAAAAEICBP8YihMe55kApM2+aMXqhK1zuKz4QU5YlfbHsOR3nvUNObo2mjTNP3IWh3dpopLM9SwmU8eZoTgyG/jB4DqRIFlHMEQCIGD5z96S1WnYWM7gZYPhOgWbWI6skAjmqKJLVSzbei3YAiBELIagot9pZSbu2eINTAw6ADMDOAyLUvsOJxCXd5QYJQEBBP0LAVJBBJgTe2Ijqfo5Xhb+bkmn9Kc3Ap1Eduw1hEqEaeNY1kG4WlvzVdVwaZ1jzeh9odRQ4ZZtl+5hqZtRmVoR636g9nRBBP2aafleMcO40sl5JeDxUd0cZ/O3Hjum248g2UTm6U2bXjrUwSpfm9bSE3z2mwFfljfeYar64p/fzPjvWkzYHjdBBP8YihMe55kApM2+aMXqhK1zuKz4QU5YlfbHsOR3nvUNObo2mjTNP3IWh3dpopLM9SwmU8eZoTgyG/jB4DqRIFlBBO6vwGIQCGYqmXe5U+5/evh5gYMPVhysUDhyEcddWLy3UUFPyavgz0oopARIBszVSz+AupkODv8Ea7G5MJaLlzpUrgABAIoCAAAAAUssX3Go9bkrmf7YBBxvMnd5t9JdhzDw2UN1KAokaWjwAQAAABcWABTpJujKuIJ6cjPFDyHuz7pUUwWWz/////8CsFMQAAAAAAAXqRRBUewMlTJ+3KHExdONAeANnsk9V4f0QqABAAAAABepFNWDZ0U79JQm9m+itiWgkI7tlDWBhwAAAABCAgT/GIoTHueZAKTNvmjF6oStc7is+EFOWJX2x7Dkd571DTm6Npo0zT9yFod3aaKSzPUsJlPHmaE4Mhv4weA6kSBZRzBEAiBjNMIz1Cab3QT5tAzCdPF/NmDEBjHMIouIIFNevRlDwwIgOSMDuyQpoURVRg5DSDSz5fSUVt+bMqUnbMNRvk7lDDwBAQT9CwFSQQSYE3tiI6n6OV4W/m5Jp/SnNwKdRHbsNYRKhGnjWNZBuFpb81XVcGmdY83ofaHUUOGWbZfuYambUZlaEet+oPZ0QQT9mmn5XjHDuNLJeSXg8VHdHGfztx47ptuPINlE5ulNm1461MEqX5vW0hN89psBX5Y33mGq+uKf38z471pM2B43QQT/GIoTHueZAKTNvmjF6oStc7is+EFOWJX2x7Dkd571DTm6Npo0zT9yFod3aaKSzPUsJlPHmaE4Mhv4weA6kSBZQQTur8BiEAhmKpl3uVPuf3r4eYGDD1YcrFA4chHHXVi8t1FBT8mr4M9KKKQESAbM1Us/gLqZDg7/BGuxuTCWi5c6VK4AAA==

Transaction fingerprint (md5):
92c22b654578cc505dc3eb6800bd8291
QR code for transaction written to transaction-01.png,transaction-02.png
First glacierscript run complete. Deleting wallet...
Bitcoin Core stopping
Running glacierscript again for the second signature...
Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?Input a filename located in the current directory which contains the PSBT:
0.07230000 BTC in unspent inputs from cold storage address 2MyCc3oEVwSYzgmspKTjZ6HZrzeg8ajdmKR
0.07226753 BTC going to destination address bcrt1qrwxllgvtc0ns624dlped894kkfq608jmye2k50
Fee amount: 0.00003247
Confirm? (y/n): 
How many private keys will you be signing this transaction with (at least 1 required to complete transaction)?
#: Key #1: 
Is this data correct?
*** WARNING: Incorrect data may lead to loss of funds ***

0.07230000 BTC in unspent inputs from cold storage address 2MyCc3oEVwSYzgmspKTjZ6HZrzeg8ajdmKR
0.07226753 BTC going to destination address bcrt1qrwxllgvtc0ns624dlped894kkfq608jmye2k50
Fee amount: 0.00003247

Signing with private keys: 
924WjH7yMYyuyhUougBHEu2vThcHRmahF6FsKvNVQBwSGuEyqKm


Confirm? (y/n): 
Calculating transaction...

Final fee rate: 1.00 satoshis per vbyte

Raw signed transaction (hex):
02000000076fe2c0e721d4d64e44684440c2cd82dc916e647b5ed06b8ff6f6f85fd7f77d6f01000000fd9f0100473044022041d7c96b123e53d4f803665a34ce57193abb926035250c79608847f1ebfae11102203bb4412f1acdef19bc91c382b815053a41a91664121bf66607e39d701e65be00014730440220404a0f818faf46307332fc5ae09ab5a2c67875f8fbf857fea72b424dd82524ad02207d95676ed0f4b75c5383e304c2f1cbfeecc29d9921a2928be90a8d3539ce597a014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffff33584c4bac82211deb288ad240296b2372014a6141dd414cc61990de29813d8e01000000fd9f01004730440220135b9de9069b74c24128da0fc095589f74327c264e852fa8a0d4d89d57a87c820220132a2f645b023da4878a929af5a97a88910bc9b3574c2ac63c565b4e2b5d98de014730440220322a979dafea81e5029c54e21058f990552e9b1462b887bf53721830209b2e75022051d8997acf6d15790508638e898392b1c63b3546365563ff7293f9e60b56e7dc014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffffee4b4352eaf79443be33c5e611758c9d3269851ca4f251dbfb43a63929c5eb1001000000fd9f01004730440220014d728e38f876072d6f838366f1dfd00ee6134f3e74ebff075caca68e2c804402204b96bff10b5977889f157356bb9781a32e6083ea199cc2c5ee66778e37f580bd0147304402200d23b9e529a22800bc5123cb774de60f674870d344db679a4e6ce24eb3928c14022068376406373b9166d863321fe6b4aa9c07fec13ad8935b47b208d00d913ee561014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffffc0f2a77bbc3ff057fe97fabdd525361a6f0e567b66400865e650f6e0820213ae00000000fd9f010047304402204d456b9ae699771515aa42e1893ace6074431e1c8c4daf26b7273eae6110ccb3022068810454aee399680f89e9452003aa042d8d555e61738a50495d6c889274166901473044022023f70c19e595b88ba7de851588678cb6b44619d3b162469540abf58af2bff68d02206b9700fd33deb30329129d1ff989e43ff457303ff6f5b89930cd79c160968c52014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffffc36a8092a910e456af8bf032640a8aee9a8748a848a0aaf115df09d110df3d8101000000fd9f010047304402200464e2eaa46ff92a937bda459ab25e6675004bc19122757b4713f894ffacb14002204101b646f4a140fd6fa6fac4fe1787b981eab971184a29ddc25b66276931b4ff0147304402204e9fb8d1fc8814c7d7e82830d9d92125eedb621d679e0b2f19c3d66aa8cf090402207e51605b321bcb1621f54a96e221673399d1ec239e16ab7ff76cebb9845fdc80014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffff8da6ee4589f8e4f76e5da7274e01fdca66d6d315eb6fcc179fc8af848fda373300000000fd9f0100473044022060f9cfde92d569d858cee06583e13a059b588eac9008e6a8a24b552cdb7a2dd80220442c86a0a2df696526eed9e20d4c0c3a003303380c8b52fb0e2710977794182501473044022020fc2b96f4a25b309fd749f4c2d8dfcf14a9fc8c05735e028a3b78c7d7cc4f4002203821d818cbc16404d85d5894ae6e8c4ed4dbabb24355ad2a875dac25dbf3c1f2014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aefffffffff8523bafbdf226c447b35b75f1f44269f2b1739af8cf476642bbd187a825363800000000fd9f010047304402206334c233d4269bdd04f9b40cc274f17f3660c40631cc228b8820535ebd1943c30220392303bb2429a14455460e434834b3e5f49456df9b32a5276cc351be4ee50c3c01473044022039ddbcb6b0f882f246447f7df059fb601684a2e5f2d31cf52effb5606a93978602201ac8a913238d73432a21c8d50722b9b9d35120827615c2101de58a52eeaf2119014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffff0181456e00000000001600141b8dffa18bc3e70d2aadf872d396b6b241a79e5b00000000

Transaction fingerprint (md5):
df27851e8de7e0bffab5e55ba4e2d9c0
QR code for transaction written to transaction-01.png,transaction-02.png
